docs/ja/openapi-fetch/api.md

```diff
@@ -180,6 +181,15 @@ const { data, error } = await client.PUT("/submit", {
 });
 ```
 
+::: tip
+
+便宜上、 `openapi-fetch` は `body` パラメーターに値が指定されたリクエストに対して、`Content-Type` を自動的に `application/json` に設定します。
```

Comment:
Just an idea to avoid duplicated "に"
```suggestion
便宜上、 `openapi-fetch` は `body` パラメーターに値が指定されたリクエストに対して、`Content-Type` を `application/json` に自動で設定します。
```